### PR TITLE
Heroku demo

### PR DESCRIPTION
**Pull request by ojhaujjwal**
- [X] Automatically deploy docker container to Heroku

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ node_modules
 /data
 npm-debug.log
 /coverage
+/demo/.env
```
Comment (geshan) [inline .gitignore]: BL

File: demo/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM mhart/alpine-node:6.8
```
Comment (geshan) [inline demo/Dockerfile]: 7.4?

File: demo/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+COPY . /var/www/paale-dai
+
+WORKDIR /var/www/paale-dai
+
+RUN apk add --no-cache curl tar && curl -o- -L https://yarnpkg.com/install.sh | sh
+ENV PATH /root/.yarn/bin:$PATH
+
+RUN chown -R root:root /var/www/paale-dai\
```
Comment (geshan) [inline demo/Dockerfile]: Nobody:nobody?

**Review by geshan — CHANGES_REQUESTED** [review]
Need some changes

File: demo/Dockerfile (hunk 2 above)
Comment (ojhaujjwal) [reply demo/Dockerfile]: +1

File: demo/Dockerfile (hunk 3 above)
Comment (ojhaujjwal) [reply demo/Dockerfile]: sorry. I didnt' get you!

Comment (geshan) [reply demo/Dockerfile]: Root:root is not ideal, either create a user or use nobody: nobody depending on Linux flavor.

File: demo/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+wget -qO- https://toolbelt.heroku.com/install.sh | sh
+heroku plugins:install heroku-container-registry
+docker build -t registry.heroku.com/paale-dai/web -f demo/Dockerfile .
+/usr/local/heroku/bin/heroku container:login
+docker push registry.heroku.com/paale-dai/web
```
Comment (ojhaujjwal) [inline demo/deploy.sh]: TODO: query health check endpoint using curl.

**Pull request merged**